iXBRLViewerPlugin/viewer/src/less/icons.less

```diff
@@ -12,8 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-.icon-search:before {
-    content: "\E986";
-    font-family: "icomoon";
+.icon-search {
+    content: "\e990";
+    font-family: "wdeskicons";
```

Comment:
Can these be nested to pick up font-family and not need to re-specify it for each icon?

Reply:
Good idea - done with a mixin in cb4f67a